src/main/java/io/vertx/rabbitmq/RabbitMQClient.java

```diff
@@ -65,26 +65,55 @@ static RabbitMQClient create(Vertx vertx, JsonObject config) {
   void basicGet(String queue, boolean autoAck, Handler<AsyncResult<JsonObject>> resultHandler);
 
   /**
+   * Use {@link RabbitMQClient#basicConsumer(String, QueueOptions, Handler)} instead
+   * <p>
    * Start a non-nolocal, non-exclusive consumer, with auto acknowledgement and a server-generated consumerTag.
    *
    * @see com.rabbitmq.client.Channel#basicConsume(String, Consumer)
    */
+  @Deprecated
   void basicConsume(String queue, String address, Handler<AsyncResult<Void>> resultHandler);
 
   /**
+   * Use {@link RabbitMQClient#basicConsumer(String, QueueOptions, Handler)} instead
+   * <p>
    * Start a non-nolocal, non-exclusive consumer, with a server-generated consumerTag.
    *
    * @see com.rabbitmq.client.Channel#basicConsume(String, boolean, String, Consumer)
    */
+  @Deprecated
   void basicConsume(String queue, String address, boolean autoAck, Handler<AsyncResult<Void>> resultHandler);
 
   /**
+   * Use {@link RabbitMQClient#basicConsumer(String, QueueOptions, Handler)} instead
+   * <p>
    * Start a non-nolocal, non-exclusive consumer, with a server-generated consumerTag and error handler
    *
    * @see com.rabbitmq.client.Channel#basicConsume(String, boolean, String, Consumer)
    */
+  @Deprecated
   void basicConsume(String queue, String address, boolean autoAck, Handler<AsyncResult<Void>> resultHandler, Handler<Throwable> errorHandler);
 
+  /**
+   * @see com.rabbitmq.client.Channel#basicConsume(String, Consumer)
+   * @see RabbitMQClient#basicConsumer(String, Handler)
+   */
+  default void basicConsumer(String queue, Handler<AsyncResult<RabbitMQueue>> resultHandler) {
+    basicConsumer(queue, new QueueOptions(), resultHandler);
+  }
+
+  /**
+   * Create a consumer with a given options.
```

Comment:
"Create a basic consumer with the given options"

Reply:
Done